### ISSUE TITLE
Timer makes redundant calls when we don't set autoPlay

### ISSUE THREAD

**Issue by MichalDanielDobrzanski** [serenader2014/flutter_carousel_slider#84]
It is because timer is scheduled with autoPlay true or false.

We should only return timer for autoPlay set to true.
I suggest to return null for timer for this case:
```
  Timer getTimer() {
    return widget.autoPlay ? Timer.periodic(widget.autoPlayInterval, (_) {
        widget.pageController.nextPage(
            duration: widget.autoPlayAnimationDuration,
            curve: widget.autoPlayCurve);
      }) : null;
  }
```

Otherwise, there are redundant anonymous function calls.

**Comment by serenader2014** [comment]
Good point. Will fix this issue in next release.

**Comment by serenader2014** [comment]
Fixed in v1.4.0
